src/styles/baseThemes/darkBaseTheme.js

```diff
@@ -19,5 +19,6 @@ export default {
     disabledColor: ColorManipulator.fade(Colors.fullWhite, 0.3),
     pickerHeaderColor: ColorManipulator.fade(Colors.fullWhite, 0.12),
     clockCircleColor: ColorManipulator.fade(Colors.fullWhite, 0.12),
+    shadowColor: Colors.fullWhite,
```

Comment:
What's your use case for this?
Here is how the `Paper` doc page looks like:
![capture d ecran 2016-01-21 a 16 08 27](https://cloud.githubusercontent.com/assets/3165635/12484163/4e0545fc-c059-11e5-8af1-ca8358ce6a8d.png)
I think that it was better in dark :grin:.